.gitignore

```diff
@@ -14,6 +14,7 @@ config.sub
 configure
 configure.in
 depcomp
+doc/
```

Comment:
There are some files under `doc/`. What about using `doc/autodocs`?

Reply:
Yes, you're right. I didn't realize about it at the time to create the PR. But then I was trying to fix it using 

```
doc/**/*.html
doc/**/*.css
doc/**/*.js
```

without success because of `Modified files not commited: doc/index.html` error.

However, thanks to your PR (#35), I'll improve this and others `.gitignores` files in the rest of projects were Travis still unhappy and there are tracked files in `doc` directory.

Thanks a lot.